index.html

```diff
@@ -20,46 +20,50 @@ <h2>New Activity</h2>
             <h3>Select a category</h3>
             <div class='category-buttons'>
               <button type="button" class="icon-study">
-                <img class='icon study' src='./assets/study.svg'>
-                <img class='icon study-active hidden' src='./assets/study-active.svg'>
+                <img class='icon study' src='./assets/study.svg' alt='study icon'>
+                <img class='icon study-active hidden' src='./assets/study-active.svg' alt='active study icon'>
                 Study
               </button>
 
               <button type="button" class="icon-exercise">
-                <img class='icon exercise' src='./assets/exercise.svg'>
-                <img class='icon exercise-active hidden' src='./assets/exercise-active.svg'>
+                <img class='icon exercise' src='./assets/exercise.svg' alt='exercise icon'>
```

Comment:
Im glad that we added the alt properties here to make this function correctly for screen readers (WAVE)